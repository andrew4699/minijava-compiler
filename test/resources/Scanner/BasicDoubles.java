double x = 5.0;

52e15
52.E15
52.5e15
.52E15

52e+15
52.E+15
52.5e+15
.52E+15

52e-15
52.E-15
52.5e-15
.52E-15

.25
.0
0.0
0.0e0
52.
