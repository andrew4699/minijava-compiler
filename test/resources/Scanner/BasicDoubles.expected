DOUBLE ID(x) BECOMES DOUBLE(5.0) SEMICOLON

DOUBLE(5.2E16)
DOUBLE(5.2E16)
DOUBLE(5.25E16)
DOUBLE(5.2E14)

DOUBLE(5.2E16)
DOUBLE(5.2E16)
DOUBLE(5.25E16)
DOUBLE(5.2E14)

DOUBLE(5.2E-14)
DOUBLE(5.2E-14)
DOUBLE(5.25E-14)
DOUBLE(5.2E-16)

DOUBLE(0.25)
DOUBLE(0.0)
DOUBLE(0.0)
DOUBLE(0.0)
DOUBLE(52.0)
